src/integration.test.ts

```diff
@@ -43,3 +43,15 @@ describe("siasky.dev end to end integration test", () => {
     );
   });
 });
+
+describe("Registry end to end integration tests", () => {
+  it("Should timeout after the given timeout time in ms", async () => {
+    const { publicKey } = genKeyPairAndSeed();
+
+    // Try getting an inexistant entry.
+    const { entry, signature } = await client.registry.getEntry(publicKey, "foo");
+
+    expect(entry).toBeNull();
+    expect(signature).toBeNull();
+  });
+});
```

Comment:
Can we add a test where we pass a `timeout` to the `set` method and see that it doesn't affect it?